lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserClang.cpp

```diff
@@ -1632,6 +1669,96 @@ DWARFASTParserClang::GetCPlusPlusQualifiedName(const DWARFDIE &die) {
   return qualified_name;
 }
 
```

Comment:
I am bothered by this name. I would expect that something called `FindDefinitionDIE` returns a DWARFDIE (or something along those lines).  Instead it returns bool, and appears to do a lot more than simply finding a DIE (it constructs the full clang ast for it, iiuc).

How about, mirroring the SymbolFileDWARF apis this wraps, we call this something like `FindDefinitionTypeForDIE` (and then actually have the function return the parsed Type object)?

Reply:
Yes, it does more than finding the DIE. Updated to return `Type*` and renamed to `FindDefinitionTypeForDIE`.